src/hooks/useMerkleVesting.ts

```diff
@@ -97,7 +102,7 @@ export const useMerkleVesting = (
 						setVestedTokens(amountVested);
 						setReleasableTokens(amountVested);
 					}
-
+					setPartialFetchToken(partialFetchToken + 1);
```

Comment:
This will only ever get set to `1`, thus being a true/false value. Why is this a number? And what purpose would this have being a number?